_topic_maps/_topic_map.yml

```diff
@@ -1017,16 +1017,18 @@ Topics:
 - Name: External DNS Operator
   Dir: external_dns_operator
   Topics:
+  - Name: Understanding the External DNS Operator
+    File: understanding-external-dns-operator
   - Name: Installing the External DNS Operator
-    File: nw-installing-external-dns-operator
+    File: nw-installing-external-dns-operator-on-cloud-providers
```

Comment:
Were the file name changed made in this PR? If so, are the absolutely necessary? And if they are necessary, have you updated the redirects register?

Reply:
Ah ok!

Reply:
@xenolinux this change does look like it removed the https://docs.openshift.com/container-platform/4.10/networking/external_dns_operator/nw-installing-external-dns-operator.html page and added https://docs.openshift.com/container-platform/4.10/networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.html in its place.

https://github.com/openshift/openshift-docs/issues/46809 relates to a 404 being returned for https://docs.openshift.com/container-platform/4.9/networking/external_dns_operator/nw-installing-external-dns-operator.html because no redirect is in place.

To resolve the issue, redirect https://github.com/openshift/openshift-docs/pull/47788 has been created.

I noticed that https://github.com/openshift/openshift-docs/blob/enterprise-4.10/networking/external_dns_operator/nw-installing-external-dns-operator.adoc still exists, although it is no longer live because it was removed from the topic map in this PR. Before we apply the redirect, can you please confirm if `nw-installing-external-dns-operator` needed to be removed and replaced by `nw-installing-external-dns-operator-on-cloud-providers`, or should it still be live? Thanks!



Before we merge that redirect PR, can you please confirm if th

Reply:
Following a discussion this morning, we have clarified that the https://docs.openshift.com/container-platform/4.10/networking/external_dns_operator/nw-installing-external-dns-operator.html should be redirected to https://docs.openshift.com/container-platform/4.10/networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.html. We are in the process of implementing that redirect through https://github.com/openshift/openshift-docs/pull/47788.

I also noticed that there are two `nw-installing-external-dns-operator.adoc` files at the moment. For example, in main:

```
$ find . -iname "nw-installing-external-dns-operator.adoc"
./modules/nw-installing-external-dns-operator.adoc
./networking/external_dns_operator/nw-installing-external-dns-operator.adoc
```

Although `./networking/external_dns_operator/nw-installing-external-dns-operator.adoc` is an assembly file path, it has a module heading. Also, the two files have some differences.

@xenolinux, as discussed, please create a PR to remove `./networking/external_dns_operator/nw-installing-external-dns-operator.adoc` and if required, update `./modules/nw-installing-external-dns-operator.adoc` to include any changes that are in `./networking/external_dns_operator/nw-installing-external-dns-operator.adoc`. Thanks!

Reply:
The PR for removing the  ./networking/external_dns_operator/nw-installing-external-dns-operator.adoc created here: https://github.com/openshift/openshift-docs/pull/47817

Reply:
Thank you.

The redirect is now in place.